store/project_member.go

```diff
@@ -84,12 +84,11 @@ func (s *Store) FindProjectMember(ctx context.Context, find *api.ProjectMemberFi
 	return projectMemberList, nil
 }
 
-// GetProjectMemberByID gets an instance of ProjectMember.
-func (s *Store) GetProjectMemberByID(ctx context.Context, id int) (*api.ProjectMember, error) {
-	find := &api.ProjectMemberFind{ID: &id}
+// GetProjectMember gets an instance of ProjectMember.
+func (s *Store) GetProjectMember(ctx context.Context, find *api.ProjectMemberFind) (*api.ProjectMember, error) {
 	projectMemberRaw, err := s.getProjectMemberRaw(ctx, find)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get ProjectMember with ID %d, error: %w", id, err)
+		return nil, fmt.Errorf("failed to get ProjectMember with projectMemberFind[%+v], error: %w", find, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to get ProjectMember with projectMemberFind %+v, error: %w", find, err)
```
Let's deprecate the usage of [] in logging message 😃